packages/katello/katello/katello-remove

```diff
@@ -172,7 +172,7 @@ end
 puts "\nWARNING: This script will erase many packages and config files."
 puts "Important packages such as the following will be removed:\n"
 puts "  * httpd (apache)"
-puts "  * mongodb"
+puts "  * rh-mongodb"
```

Comment:
I'd leave this as mongo. These are generic service names rather than actual package names.